File: programme/minutes-to-seconds/minutes-to-seconds.c
// Write a programme to convert minutes to seconds
#include <stdio.h>

int main(void)
{
    double minutes;
    printf("Min : ");
    scanf("%lf", &minutes);
    int seconds = minutes * 60;
    printf("Sec : %d\n", seconds);
    return 0;
}

File: programme/minutes-to-seconds/minutes-to-seconds.go
// Write a programme to convert minutes to seconds

package main

import (
	"fmt"
	"strconv"
)

func main() {
	var str string

	fmt.Print("Input Minutes : ")
	fmt.Scan(&str)

	min, err := strconv.Atoi(str)
	if err != nil || min < 0 {
		fmt.Printf("Input error: %s", str)
		return
	}

	fmt.Printf("\n%d seconds\n", min*60)
}